packages/legacy/core/App/screens/UseBiometry.tsx

```diff
@@ -283,7 +283,7 @@ const UseBiometry: React.FC = () => {
         {biometryAvailable ? (
           <View>
             <Text style={TextTheme.normal}>{t('Biometry.EnabledText1')}</Text>
-            <Text></Text>
+            <Text accessible={false}></Text>
```

Comment:
if these aren't rending text we should just remove them and add padding if we need the space